cgp/population.py

```diff
@@ -128,3 +128,16 @@ def fitness_parents(self) -> List[Union[None, float]]:
             List of fitness values for all parents.
         """
         return [ind.fitness for ind in self._parents]
+
+    def reorder_genome(self) -> None:
+        """ Reorders the genome for all parents in the population
+
+        Creates a neutral drift though the search space,
+        by reordering parents and not offspring.
```

Comment:
i'm not sure we should add that information here. in general, reordering should be documented at one point, not in different parts, like `hl_genome` and `Population`

Reply:
I think you are right.  I moved the hl-description into `MuPlusLambda.step` and the details are still in `genome.reorder`, please tell me if think this is appropriate.